meta.jq

```diff
@@ -151,63 +133,50 @@ def build_command:
 	normalized_builder as $builder
 	| if $builder == "buildkit" then
 		git_build_url as $buildUrl
-		| (
-			(should_use_docker_buildx_driver | not)
-			or docker_uses_containerd_storage
-		) as $supportsAnnotationsAndAttestsations
 		| [
 			(
 				[
 					@sh "SOURCE_DATE_EPOCH=\(.source.entry.SOURCE_DATE_EPOCH)",
 					# TODO EXPERIMENTAL_BUILDKIT_SOURCE_POLICY=<(jq ...)
 					"docker buildx build --progress=plain",
-					if $supportsAnnotationsAndAttestsations then
-						"--provenance=mode=max",
+					"--provenance=mode=max",
+					if build_should_sbom then
 						# see "bashbrew remote arches docker/scout-sbom-indexer:1" (we need the SBOM scanner to be runnable on the host architecture)
 						# bashbrew remote arches --json docker/scout-sbom-indexer:1 | jq '.arches | keys_unsorted' -c
 						if .build.arch as $arch | ["amd64","arm32v5","arm32v7","arm64v8","i386","ppc64le","riscv64","s390x"] | index($arch) then
 							# TODO this needs to be based on the *host* architecture, not the *target* architecture (amd64 vs i386)
 							"--sbom=generator=\"$BASHBREW_BUILDKIT_SBOM_GENERATOR\""
-							# TODO this should also be totally optional -- for example, Tianon doesn't want SBOMs on his personal images
```

Comment:
This design resolves this `TODO` nicely because I'll just patch out `doi.jq` in favor of a `tianon.jq` that always returns `false` for my personal builds :+1: